tests/lib.rs

```diff
@@ -21,7 +21,15 @@
 
 extern crate self_encryption;
 extern crate rand;
+extern crate rustc_back;
```

Comment:
Those MIT docs are old, you should use the officially hosted ones, tempdir is now http://doc.rust-lang.org/std/fs/struct.TempDir.html